DEVELOPMENT.md

```diff
@@ -68,16 +66,6 @@ client.
 Example linux env below:
 
 ```
-export DEADLINE_CLOUD_PYTHONPATH="/tmp/lib/python3.11/site-packages"
+export C4DPYTHONPATH311 "/path/to/deadline-cloud/site-packages"
 export DEADLINE_CINEMA4D_EXE="/opt/maxon/cinema4dr2024.200/bin/c4d"
-```
-
-### Windows
```

Comment:
Hmm, so we don't need to manually install pywin32 anymore? But wouldn't this raise module not found errors due to the adaptor?

Reply:
when I pip install deadline-cloud-for-cinema-4d to a interpreter search path supported by Cinema 4D it installs the pywin32 dependency too, both submitter and adaptor load it

Reply:
Hmm, I've had issues with this error when I don't install pywin32 separately. But I'll investigate this on my side. Thanks for the tip. 

```
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)Traceback (most recent call last):
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)  File "C:\ProgramData\Amazon\OpenJD\session-1e008b980032473da35ef8f59bb9942e09l386vr\.env\Lib\site-packages\deadline\cinema4d_adaptor\Cinema4DClient\plugin\DeadlineCloudClient.pyp", line 25, in <module>
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)    from deadline.cinema4d_adaptor.Cinema4DClient.cinema4d_client import main
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)  File "C:\ProgramData\Amazon\OpenJD\session-1e008b980032473da35ef8f59bb9942e09l386vr\.env\Lib\site-packages\deadline\cinema4d_adaptor\Cinema4DClient\__init__.py", line 3, in <module>
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)    from .cinema4d_client import Cinema4DClient
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)  File "C:\ProgramData\Amazon\OpenJD\session-1e008b980032473da35ef8f59bb9942e09l386vr\.env\Lib\site-packages\deadline\cinema4d_adaptor\Cinema4DClient\cinema4d_client.py", line 9, in <module>
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)    from openjd.adaptor_runtime_client import (
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)  File "C:\ProgramData\Amazon\OpenJD\session-1e008b980032473da35ef8f59bb9942e09l386vr\.env\Lib\site-packages\openjd\adaptor_runtime_client\__init__.py", line 19, in <module>
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)    from .win_client_interface import WinClientInterface as ClientInterface  # type: ignore
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)  File "C:\ProgramData\Amazon\OpenJD\session-1e008b980032473da35ef8f59bb9942e09l386vr\.env\Lib\site-packages\openjd\adaptor_runtime_client\win_client_interface.py", line 12, in <module>
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)    from .named_pipe.named_pipe_helper import NamedPipeHelper
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)  File "C:\ProgramData\Amazon\OpenJD\session-1e008b980032473da35ef8f59bb9942e09l386vr\.env\Lib\site-packages\openjd\adaptor_runtime_client\named_pipe\named_pipe_helper.py", line 7, in <module>
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)    import win32file
2024/10/25 07:48:35+00:00 ADAPTOR_OUTPUT: STDOUT: (null)ModuleNotFoundError: No module named 'win32file'
```